kin-core/src/main/java/kin/core/KinClient.java

```diff
@@ -30,12 +30,13 @@
      * KinClient is an account manager for a {@link KinAccount}.
      *
      * @param context the android application context
-     * @param provider the service provider provides blockchain network parameters
+     * @param id unique id for identifying this client, different ids will store a different accounts
+     * @param provider the service provider - provides blockchain network parameters
      */
-    public KinClient(@NonNull Context context, @NonNull ServiceProvider provider) {
+    public KinClient(@NonNull Context context, @NonNull String id, @NonNull ServiceProvider provider) {
```

Comment:
Does `id` should optional and use default? 
It's not a must for every digital service right?

Reply:
rename to `storageKey`